src/RobotMap.h

```diff
@@ -19,10 +19,10 @@ class RobotMap {
   static const int kIDShooterLeft = 2;
   static const int kIDIntakeRight = 3;
   static const int kIDIntakeLeft  = 4;
-  static const int kIDBallConveyorLeft = 5;
-  static const int kIDBallConveyorRight = 6;
+  static const int kIDFeederLeft = 5;
+  static const int kIDFeederRight = 6;
```

Comment:
Do we want an ID naming convention that includes the subsytem name? This would be like RobotBuilder names motors.